cityofaustin/atd-moped
# Adds milestone edit modal

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/9667

## Testing
**URL to test:** 
https://deploy-preview-769--atd-moped-main.netlify.app/

**Steps to test:**
go to dashboard and click the status badge or milestones meter for a modal to update phases or milestones, respectively

there is a minor issue where there is a delay from when the modal is opened to when the table is displayed. fixing this will require refactoring the timeline component. i will make an issue to include this refactor in an upcoming release.

---
#### Ship list
